lib/ace/ext/papeeria/tex_completer.coffee

```diff
@@ -252,6 +255,7 @@ define((require, exports, module) ->
       constructor: ->
         @refCache = new CompletionsCache(getJsonFromUrl, processReferenceJson)
         @citeCache = new CompletionsCache(getJsonFromUrl, processCitationJson)
+        @filesCache = new CompletionsCache(getJsonFromUrl, processFilesJson)
```

Comment:
If `processFilesJson` is not supposed to do anything other than returning its argument, you may want to replace it with `(json) -> json`